Von Neumann Class - a mid-pop solgov ship

Pockets-byte opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

I've been playing NSV on and off recently, and many of the ships we play on have some glaring issues- the first of which being that they're all... Just. Too. Big. 
the first goal of this ship is to reach near full functionality in less space use, to allow crew function more effectively during combat and out of it- in this it currently is a little over half the size of the atlas while still mounting most of what I'd consider an effective ship to need. secondly, the other solgov ship currently in rotation suffers from a serious lack of redundancy, and is frankly too big for it's crew complement, the von Neumann was designed almost from the ground up with redundancy and damage control in mind, although this has been toned down. in addition, solgov ships suffer from muni being overly simplified, which this ship also attempts to address via the inclusion of a full VLS system and a strong emphasis on muni-operated laser PDCs, as well as generally requiring at least one person per weapon
finally this ship does nothing to singularity engines =(

the ship itself is powered by an overclocked ANGCR, and has a disconnected TEG for people who want even more power. it also mounts a shield, and has provisioning mostly inspired or copied from the wisp. for armament, there are six laser PDCs, a small full VLS system with 8 silos. a heavy LAMS system, and two burst phasers. it has no heavy weapons, and will be the most fragile ship in the game comparatively, but the powerful reactor and high maneuverability make up for that 
most departments exist in some form on this ship, although a few things have been cut in the name of compactness, there is no church or chaplain, no lawyer, no viro, brig physican, deputy, or ATC. in keeping with the star-treky theme of solgov, the ship does mount replicators in a couple places, although there is both botany and a kitchen-bar. 
maint was initially overlooked, but has some presence. and the there are two hangars and a full turing drive as with respects to the ship's higher crew requirements.
 the ship's sprite is also fully animated, and has semi-realistic navigation lights


description very out of date
## Why It's Good For The Game

more solgov ships good. more energy weapon propaganda, small ship that shouldn't be completely awful. 

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

![image](https://github.com/BeeStation/NSV13/assets/55299415/f6f8f141-7aa4-44ea-920b-f42286ec2b9a)
![image](https://github.com/BeeStation/NSV13/assets/55299415/64a3e238-da37-4e7e-883f-a176112fd3ef)
![image](https://github.com/BeeStation/NSV13/assets/55299415/bb99b7a1-fc30-4e26-8183-b75767e51e06)
![destroyer](https://github.com/BeeStation/NSV13/assets/55299415/ce84788a-5a60-4826-b375-760aac9cd4e9)
![frigate](https://github.com/BeeStation/NSV13/assets/55299415/20895260-2d56-4a5c-a6a6-99104fd73601)


https://github.com/BeeStation/NSV13/assets/55299415/4080afcf-42f2-4fd0-bf8b-2faa735139ca
![ship](https://github.com/user-attachments/assets/bd8d738b-42d9-46a7-9f6d-81a911bdaba9)




</details>

## Changelog
:cl:
add: a new shipmap
add: two new solgov ship-sprites
add: a new AI ship for solgov, the VN class frigate, a lighter ship mounting no heavy weaponry, but carries torpedos making it just as dangerous as any other solgov ship
tweak:solgov interdictor has it's own sprite
tweak:changed twoish laser descriptions slightly to respect the setting more
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Pockets-byte committed:
Added map

Pockets-byte committed:
tons of work on the map, and touching rad collectors to make singularity and other radiation based engines more viable

push you idiot, also rad collector stats are not touched

Pockets-byte committed:
more work is done

Pockets-byte committed:
Update .gitignore

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
more stuff

Pockets-byte committed:
names and defunct roof removed

Pockets-byte committed:
work maybe?

Pockets-byte committed:
fixes space in walls

Pockets-byte committed:
more work

I have a headache yay. I am slowly losing it

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
fixes le stupid

Pockets-byte committed:
AAAAaaaaaaaaaa

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte committed:
moved the icon files

hweeeeeeeeeeeeeeeeeeeeeeeeeeeee. help

DuskLight9978 commented:
Ping me to this PR when the ships complete as it regards SOLGOV ships

SerynEngi commented on _maps/Vonneumann.json:
```diff
@@ -0,0 +1,34 @@
+    "manufacturer": "[Redacted]",
+	"mapper": "Truncated-Cake",
+    "commission_date": "May 19, 2264",
+    "strengths": ["Legal energy shields", "Energy weapons", "Compact and easy to repair", "High tech,low maintenance singularity engine", "Highly Manuverable", "Advanced technology", "Smallest ship to ever mount a Turing Drive" ],
```
Codewise, there is no such thing as "Legal" energy shields. There are simply energy shields or not. Also, misspelling.

Pockets-byte replied:
was like this on the wisp. will fix

Pockets-byte replied:
legal energy shields do in fact exist, for any solgov ships, which this one is, so that'll be going back in the description

SerynEngi commented on _maps/Vonneumann.json:
```diff
@@ -0,0 +1,34 @@
+    "weaknesses": ["Fragile", "Power-hungry", "Near Non-existent armor", "Higher crew requirements", "Extremely small" ],
+	"equipment": [
+		"SolGov shield generator",
+		"1 NT-ST049 'Sturm' coaxial railgun",
```
Get approval from the leads before the use of this weapon.

SerynEngi commented on code/modules/power/singularity/singularity.dm:
Please explain why you changed this.

Pockets-byte replied:
this is a bug that causes singularities to explode prematurely, I had thought the fix was already merged

SerynEngi commented on _maps/Vonneumann.json:
```diff
@@ -0,0 +1,34 @@
+		"Triple Laser Anti Missile System",
+		"Defunct Missile Array",
+		"2 light fighters",
+		"2 heavy fighters",
```
This fighter complement, in general, is very heavy for a ship of the intended size. In addition. Heavy fighters are uncommon by design.

SerynEngi commented on nsv13/code/modules/overmap/types/solgov.dm:
```diff
@@ -56,6 +56,14 @@
 	starting_system = "Staging" //Required for all player ships
 	armor = list("overmap_light" = 99, "overmap_medium" = 50, "overmap_heavy" = 25)
 
+/obj/structure/overmap/nanotrasen/solgov/vnc
```
You have not added the mas of the ship or the bound height/width. This is important to determining the hitbox and based on your description, this ship is supposed to be smaller and faster.

Pockets-byte committed:
Removes everything bad

I'm finishing this up

Pockets-byte committed:
bleh

I just want this over at this point

Pockets-byte committed:
stuff

Pockets-byte committed:
works ingame now

Pockets-byte committed:
A lot of stuff

Pockets-byte committed:
the shields are legal

SerynEngi commented on _maps/map_files/vonneumann/vnmk3.dmm:
Request you change this map to comply with the engineering section of the NSV Mapping Standards listed here: https://docs.google.com/spreadsheets/d/1UMT2l0rwiBcvEWb4hHBH3IejQS4RLkcZqQzvvMnWBOc

Your request for an exception (to use the Singularity and/or TEG) as a primary power supply has been denied by @Bokkiewokkie and myself.

Pockets-byte replied:
the ship should more or less confine to engineering standards barring a few exceptions now

Pockets-byte committed:
goodnight

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte commented:
I don't want to make a second wisp.

Pockets-byte commented on code/modules/power/singularity/collector.dm:
done. thank you for wasting my time and effort

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte commented:
I was a little premature with that, nerves are getting a little thin at times

Pockets-byte committed:
e.e

replaces the engine with the nuclear reactor. for now
L E N G T H E Ns the ship and readds the turing drive
makes the ship a little easier to see without aimmode on.
adds a 4th lams to make it square
gives me a headache

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte committed:
STOP CAPITALIZING MY JSONS

Pockets-byte committed:
No more capitalized jsons

Pockets-byte committed:
Update vnmk3.dmm

Pockets-byte committed:
Update vnmk3.dmm

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte committed:
Update vnmk3.dmm

Pockets-byte committed:
wheeeeeeeeeeeeeeeeeeeeeeeeeeeeee

Pockets-byte committed:
Update vonneumann.json

Pockets-byte committed:
Update vnmk3.dmm

Pockets-byte committed:
one additional solgov AI ship, new sprite for the interdictor. more work on the map

Pockets-byte committed:
errors fixed yay

Pockets-byte committed:
standarzisation

Pockets-byte committed:
departments up to code

Pockets-byte committed:
getting much closer to done

Pockets-byte committed:
made some item descriptions make more sense.

really bugged me

Pockets-byte committed:
most of everything else

Pockets-byte committed:
oops. fixed wrong value in collector

Pockets-byte committed:
removes sprite source files, fixes overlaps I forgot

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte committed:
missed one =(

Pockets-byte commented:
should be up to spec* now, assuming I didn't forget anything, in most cases.

Pockets-byte committed:
assuming this doesn't break everything, maybe this'll make the map compile?

Pockets-byte committed:
Adds job restrictions, and some really bad ascii art

Pockets-byte committed:
AAAAAAAaaaaaa

Pockets-byte committed:
more problemfixing

Pockets-byte committed:
work

Pockets-byte committed:
AI nodes theoretically work

Pockets-byte committed:
flags, I think

Pockets-byte committed:
overlap be gone

Pockets-byte committed:
missing "

Pockets-byte committed:
blah

Pockets-byte committed:
forgot to do areas

Pockets-byte committed:
makes the ship a bit weaker, and the armor values match the wisp

DuskLight9978 commented on _maps/map_files/vonneumann/vnmk3.dmm:
Error at line 4453 of this file, Likely missing "" at a specific variable of the map

Pockets-byte replied:
I think it should be fixed now

Pockets-byte committed:
progress

Pockets-byte committed:
Create vonneumann.dm

Pockets-byte committed:
head hrut

Pockets-byte committed:
fixes atmos

Pockets-byte committed:
missing quotation marks =/

Pockets-byte committed:
removes redundancy(mostly)

=(

Pockets-byte committed:
wiring

Pockets-byte committed:
shutters yaaaay

Pockets-byte committed:
access changes... maybe

Pockets-byte committed:
overlaps yay!

Pockets-byte committed:
theoretically everything else

Pockets-byte committed:
= 0 no more

Pockets-byte committed:
single period

Pockets-byte committed:
adds an exterior button to the hangar

Pockets-byte committed:
more  moving individual wires and pipes

Pockets-byte committed:
bleh

Pockets-byte committed:
vent

Pockets-byte committed:
Update vnmk3.dmm

Pockets-byte committed:
:x:

Pockets-byte committed:
flooooooors

Pockets-byte committed:
additional work

Pockets-byte committed:
work's done for now

Pockets-byte committed:
TEG's dead for now

Pockets-byte committed:
pop limits arbitrarily changed

Pockets-byte committed:
actually gave the ai vnc lasers

Pockets-byte committed:
lots of work

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte committed:
OVERLAP BEGONE

Pockets-byte committed:
bleh

Pockets-byte committed:
wallmounted pain

Pockets-byte committed:
AAAAAAAAAAAA

Pockets-byte committed:
forgot one last thing

Pockets-byte committed:
pete

Pockets-byte committed:
the ship gained weight

Pockets-byte committed:
forgot the wirenodes

Pockets-byte committed:
area stuff

Pockets-byte committed:
theoretically last of the pipes

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
last one for a while

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte committed:
"SSSSS"

Pockets-byte committed:
I swear this is the last fix

Pockets-byte committed:
late additions

Pockets-byte committed:
I forgot the "s on a button

Pockets-byte committed:
things I guess

Pockets-byte committed:
adds a playable solgov fighter, a fighter lasercannon, and maps them into the ship

Pockets-byte committed:
Revert "adds a playable solgov fighter, a fighter lasercannon, and maps them into the ship"

This reverts commit a0887c00872fe3dd00fbb50e71694b45c674339c.

Pockets-byte committed:
fixing more standout issues and areas

Pockets-byte committed:
cleaning

Pockets-byte committed:
debugging

Pockets-byte committed:
light switches mostly

Pockets-byte committed:
more polish

Pockets-byte committed:
oxygen tanks

Pockets-byte committed:
description stuff

Pockets-byte committed:
description AAAAA

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Bokkiewokkie reviewed (changes requested):
I looked over the ship, and here are some things I noticed in the mapping:
Botany can't reach half of their machines (you can't reach over trays)
The ORM is in the hangar instead of cargo
There are two mining point vendors, one of which is blocked by two suit storage units (you can just remove it? This ship also doesn't *need* four miners, so you could consider removing a job slot)
You are allowed to stack the freight torps to take up less space (they are currently blocking the freight tube console)
This ship has plenty of space near Cargo for a cargo shuttle, so please use one instead of the Atlas freight elevator.
Please don't put hull tiles below the big (front) engines, they cannot be repaired there. We also do not use those old engine block objects you put there, so please remove them.
The AI section does not need a whole cyborg recharging room, there are a lot of cyborg chargers in the hallway already. (please put some rechargers around the rest of the ship?)
The crew should not be expected to walk through the hangar to reach a bathroom or shower, they should be easily accessible.
A camera and lamp are on the same wall spot in the central starboard hallway.
You do not need to place zebra interlock points on windows, these are spawned automatically.
You DO need to place zebra interlock points on doors and directionally windowed counters (also don't forget to add directional firelocks to them), specifically talking about medical & the bar.
Medical is currently very big and open, consider adding some firelock points for fire safety (and remember to add fire alarms for seperated sections).
There is a corner wall missing in genetics, causing a diagonal disconnect (you do not need two geneticists on such a small ship, if you want to save space).
The chemists cannot reach the counter next to their chem machine, and neither can medical reach the laptop on their counter (unless that's just fluff?).
The CMO office does not need two entrances, if you want to save space to move that fire alarm onto an unoccupied wall instead of under the air alarm.
The hallway being two tiles wide is already pretty tough, but adding fire and emergency lockers which block half of the path outside of cryo is even worse, consider taking some space from xenobio's bottom left to give them a little spot inside the wall.
It seems like you added an accidental south-all-access marker and pod docking point to the starboard aft airlock, if you do want to make this a docking point, please remove the armour plates.
There is no centcom shuttle docking point.
The atmospherics and reactor rooms are pretty big, consider adding some firedoor points to compartmentalize them.
EVA has no RCD and magboots, this is not required but a standard pretty much everwhere. You also cannot insert objects into SSUs by mapping them on top of the SSU.
The HoS has a request console in the middle of their office, instead of on the wall.
The armory has no fire or air alarms
The shutters on the windows of the Detective's office are vertical instead of horizontal.
You cannot reach the bank safe in the vault, as it is blocked by the IFF console and nuclear device. Please also move the gold into a storage object that can actually be opened normally (e.g. vault freezer).
The camera in the bottom left of the torp factory room is on the same tile as a lamp (and the entire factory area could use some better light coverage, I think.
That extra bit of vault area (entrance airlock?) doesn't do much outside of adding one extra door. You could use some of it to place extra lockers/vending machines instead of having them block the hallway.
The bathroom stalls need door bolt buttons and lightbulbs, and one of them should be a cyborg recharging station.
This janitorial cupboard is sadder than the one on the Atlas, and I didn't even know that was possible. Please give the janitor a key that actually makes it possible to ride the wagon.
The portside airlocks do not have lighbulbs in them.
The hangar window has a camera inside of it.
The hangar should contain at least one sabre, for recovery and boarding missions.
The shield generator should not be accessible by science, or vice versa. That connecting engineering door seems pointless. (and removing it will give you space to take that fire alarm off the window, where it cannot be interacted with.

Pockets-byte committed:
Oops

Pockets-byte committed:
pain. 90% pain and 10% fixing

Pockets-byte committed:
headache

Pockets-byte committed:
cleanup and cramming a shuttle in somehow

Pockets-byte committed:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Pockets-byte committed:
cargo shuttle

Pockets-byte committed:
guidelines

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
radiation shielding for the reactor, fixed airlocks. flavor warning, and a slight nerf to AI shieldgen

Pockets-byte committed:
revamped science area

Pockets-byte committed:
areas

Pockets-byte committed:
slight power rerouting

Pockets-byte committed:
minor layering and a few other tweaks

Pockets-byte committed:
cleanup

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
fixes missing asteroid protection on a portion of starboard maints

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
touchup

Pockets-byte committed:
Removes shieldwall IDs

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
minor changes. mostly just floor stuff.

Bokkiewokkie reviewed (changes requested):
Code only review, here's some stuff I still saw

Pockets-byte commented on nsv13/code/modules/power/supercollector.dm:
```diff
@@ -0,0 +1,20 @@
+		/obj/item/stock_parts/manipulator = 1,
+		/obj/item/stack/sheet/mineral/uranium = 5,
+		/obj/item/stack/ore/bluespace_crystal = 5)
+	needs_anchored = FALSE
```
I initially left it in for future use, because without this, radiation engines are unusable again, but I can still remove it

Pockets-byte commented on _maps/vonneumann.json:
*headdesk

Pockets-byte commented on config/maps.txt:
```diff
@@ -63,6 +63,12 @@ map snake
 endmap
 
+map vonneumann
+	minplayers 3
```
there's some issue here though. this ship probably needs at least 2-3 people. even if you juggle tac and helm, you only have missiles on tac, and pulse phasers on helm. I guess you could theoretically move helm into gunnery, so I'll do that

Pockets-byte committed:
Apply suggestions from code review

Co-authored-by: Bokkiewokkie <43698041+Bokkiewokkie@users.noreply.github.com>

Pockets-byte committed:
Update vnmk3.dmm

funnyfreeperson commented on _maps/map_files/vonneumann/job_changes.dm:
```diff
@@ -0,0 +1,51 @@
+	..()
+	MAP_JOB_CHECK
+	total_positions = 6
+	spawn_positions = 6
```
I don't expect the muni-techs to be filled up to be perfectly honest. then again. 
MAA is technically intended to handle the missiles

Pockets-byte committed:
hindsight is 20/20

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
last reference

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Bokkiewokkie reviewed (changes requested):
An updated mapping review with my notes:
The centcom ferry docking point needs more space (the cc shuttle is 5 tiles wide)
The ORM needs two access points, one side to input ores, and one side which can output ores (usually the opposite side)
The button for the mining hangar should probably be on the other side, putting it behind a closet is a bit weird.
The mulebot in cargo cannot get out on its own right now if anything is in the storage spot, consider putting its stationed point somewhere more accessible.
The choice to have a full cargo shuttle inside the ship is weird to me. Cargo is very close to space, and if you just let it dock from outside you have a ton more space inside, perhaps you can even add a QM office then.
There's a fire alarm on the same wall as a camera in the AI upload, in the room next to it, and there is a lamp on a fire alarm in the hallway below it.
There are still zebra interlock helpers on the windows everywhere.
This ship really does not need a Thirring Drive, since it's pretty small.You also don't *need* to install four pylons for the engine to work, for a smaller ship having a few less could make saving space easier at the cost of efficiency.
There is a camera on a lamp in telecomms, and a lamp on a window in tech storage (where you already have a lamp anyway in the top right).
There are two intercoms in the CE office.
There's an intercom on the same wall as a camera in atmospherics.
There's cameras on an APC, wall medivend, and fire alarm in medical. There is also a camera on a lamp in chemistry.
There's an air alarm on a window in the hallway outside of medical (consider putting it on the wall south of it)
The floor markings in the main hallway are inconsistent between the forward port and starboard for the same structural turn and it's slightly upsetting.
There is a camera and lamp on a door into the armor pump room.
There is a turret control unit on the floor in the vault
There's a requests console in the vault wall on the right
There are two air alarms and a camera on the floor in the shieldgen room
There is a camera on the outside button to the hangar door, maybe move the button.
There is a lamp on a fire alarm in munitions
Please make the nucleium tank injector pipe go through the FTL room instead of EVA, engineers do not have default access there.
The FTL room does not have a fire alarm
You need to manually set the `department`, `department type`, and `id` for every requests console (you can look in the code and on other maps for references) they do not set themselves.

Pockets-byte committed:
ferry space

Pockets-byte committed:
fixes most other issues

Pockets-byte committed:
request console hell

Pockets-byte committed:
fixes pixel_y = 0

Pockets-byte committed:
fixed cargo windoor access. etc

added a tiny bit of protection to the ore silo. a little bit of mostly aesthetic "protective" glass on the reactor console. made the QM's closet not be in front of the ORM, stuff like that

Pockets-byte committed:
complete cargo redesign

Pockets-byte committed:
not sure what changed?

Pockets-byte committed:
no more spirited away starring cargo shuttle

Pockets-byte committed:
forgot ammo for the hangar

am dum

Pockets-byte committed:
fixes some weirdness from various changes

I gotta fix the durn doors next

Pockets-byte committed:
fixes a number of door permissions

Pockets-byte committed:
hangar signs

Pockets-byte committed:
I forgot already

Pockets-byte committed:
moved the Chrimas tree

Pockets-byte committed:
slightly tweaks the lore paeg to be a little more targeted

Pockets-byte committed:
slight lore tweak

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
minor laser lore tweaks

Bokkiewokkie reviewed (changes requested):
Only specific things for requests are what I marked in the code. I don't see any functional issues on the map outside of things I have previously mentioned (e.g. tight hallways)
For me to approve it, though, I'd like you to justify why this ship needs a complete Thirring drive and AGCNR, while it is supposedly a small and simple ship (as both of these things make it more complex and require more crew)

Pockets-byte commented:
> Only specific things for requests are what I marked in the code. I don't see any functional issues on the map outside of things I have previously mentioned (e.g. tight hallways) For me to approve it, though, I'd like you to justify why this ship needs a complete Thirring drive and AGCNR, while it is supposedly a small and simple ship (as both of these things make it more complex and require more crew)

the AGCNR is actually the only engine that can meet the energy requirements of this ship as far as I know- and at least with my engineering skills, it still needs a significant fuel use rate to power the ship. I don't remember the exact stats, but I believe the overall power draw is actually higher than the wisp for example with the weapons and shields charged. 

the thirring drive is there, partially because I prefer it over the FTL console, and because it lines up with the population guidelines in the requirements- this ship was designed to be small and compact, but not necessarily low pop- you  could theoretically fly it with one person, but I don't believe you'd see any actual capability that way. to actually see functionality out of this ship, you need at least 4+ people, and ideally 9-10+ people to really get your money's worth. 
I think a full crew could well be 15+ people, but I'm not sure where the real cutoff for crew count is. 
if you want I can replace the second pylon set with more maint-space

Pockets-byte committed:
Apply suggestions from code review

Co-authored-by: Bokkiewokkie <43698041+Bokkiewokkie@users.noreply.github.com>

Pockets-byte committed:
nukes the description

Pockets-byte commented on _maps/vonneumann.json:
is this any better?

Bokkiewokkie replied:
Much! 👍

Bokkiewokkie commented:
> > Only specific things for requests are what I marked in the code. I don't see any functional issues on the map outside of things I have previously mentioned (e.g. tight hallways) For me to approve it, though, I'd like you to justify why this ship needs a complete Thirring drive and AGCNR, while it is supposedly a small and simple ship (as both of these things make it more complex and require more crew)
> 
> the AGCNR is actually the only engine that can meet the energy requirements of this ship as far as I know- and at least with my engineering skills, it still needs a significant fuel use rate to power the ship. I don't remember the exact stats, but I believe the overall power draw is actually higher than the wisp for example with the weapons and shields charged.
> 
> the thirring drive is there, partially because I prefer it over the FTL console, and because it lines up with the population guidelines in the requirements- this ship was designed to be small and compact, but not necessarily low pop- you could theoretically fly it with one person, but I don't believe you'd see any actual capability that way. to actually see functionality out of this ship, you need at least 4+ people, and ideally 9-10+ people to really get your money's worth. I think a full crew could well be 15+ people, but I'm not sure where the real cutoff for crew count is. if you want I can replace the second pylon set with more maint-space
> 
> I wasn't really aiming for simplicity here honestly. small yes, simple, not really.

Sorry for the late reply.
Regardless, the SD is designed with a specific solgov variant to power ships like these better. You can use that, or even upgrade it a bit yourself too (within reasonable limits, it doesn't *need* to always run the shields at 100% capacity on a normal mix either, and remember that in a good situation, if the shields are enabled the armor pumps can be disabled for extra power, adding more unique dynamics to save power too!)

Secondly, if the *weapon power draw* is your issue, maybe *turn off or remove* some of the weapons? The whisp is already incredibly powerful, so if you are using more power than it, that's probably a sign you will be fine after removing a few!

Your personal preference for an FTL drive, in this case, does not seem like a good reason to me. But because you have mentioned the ship is not for lowpop as the second, I will then be raising the player limit for the map to at least 15 as well (to fit the requirements in the mapping guidelines, as you referenced). We will hopefully be making the "regular" FTL drive more interesting in the future as well, though right now it is obviously still as it remains.

Replacing some of the pylons still seems like a fine idea to me too, that's up to you, though.

Pockets-byte commented:
> > > Only specific things for requests are what I marked in the code. I don't see any functional issues on the map outside of things I have previously mentioned (e.g. tight hallways) For me to approve it, though, I'd like you to justify why this ship needs a complete Thirring drive and AGCNR, while it is supposedly a small and simple ship (as both of these things make it more complex and require more crew)
> 
> > 
> 
> > the AGCNR is actually the only engine that can meet the energy requirements of this ship as far as I know- and at least with my engineering skills, it still needs a significant fuel use rate to power the ship. I don't remember the exact stats, but I believe the overall power draw is actually higher than the wisp for example with the weapons and shields charged.
> 
> > 
> 
> > the thirring drive is there, partially because I prefer it over the FTL console, and because it lines up with the population guidelines in the requirements- this ship was designed to be small and compact, but not necessarily low pop- you could theoretically fly it with one person, but I don't believe you'd see any actual capability that way. to actually see functionality out of this ship, you need at least 4+ people, and ideally 9-10+ people to really get your money's worth. I think a full crew could well be 15+ people, but I'm not sure where the real cutoff for crew count is. if you want I can replace the second pylon set with more maint-space
> 
> > 
> 
> > I wasn't really aiming for simplicity here honestly. small yes, simple, not really.
> 
> 
> 
> Sorry for the late reply.
> 
> Regardless, the SD is designed with a specific solgov variant to power ships like these better. You can use that, or even upgrade it a bit yourself too (within reasonable limits, it doesn't *need* to always run the shields at 100% capacity on a normal mix either, and remember that in a good situation, if the shields are enabled the armor pumps can be disabled for extra power, adding more unique dynamics to save power too!)
> 
> 
> 
> Secondly, if the *weapon power draw* is your issue, maybe *turn off or remove* some of the weapons? The whisp is already incredibly powerful, so if you are using more power than it, that's probably a sign you will be fine after removing a few!
> 
> 
> 
> Your personal preference for an FTL drive, in this case, does not seem like a good reason to me. But because you have mentioned the ship is not for lowpop as the second, I will then be raising the player limit for the map to at least 15 as well (to fit the requirements in the mapping guidelines, as you referenced). We will hopefully be making the "regular" FTL drive more interesting in the future as well, though right now it is obviously still as it remains.
> 
> 
> 
> Replacing some of the pylons still seems like a fine idea to me too, that's up to you, though.
> 
> 
> 
> 
> 
> 

The reason the weapons on this ship out-draw the wisp is because the main armament is entirely a large number of secondary weapons- the phase cannon absolutely draws more individually. Then again the VLS system does exist for a theoretically heavier hit than the weapons provide. 
I'll be honest my worry is not that the ship will be overpowered, it'll be that it'll be underpowered- the laser point defense is not what I would call an overpowering weapon. But if you think that the armament is excessive, I can decrease it, I'm not entirely sure how to make a type V produce the something like 40-60 gw to power this ship though. And yes, power comprises can be made, but the ship is also made of tissue paper. I'll look at the code and mapping for the mark V, I'm more than a little nervous about that though(and am kinda not wanting to completely redesign engineering completely a third time) - I'll get back to you on that after I've done some research. 
As for the Thirring drive I'll shrink it

Pockets-byte committed:
shrinks the thirring drive

Pockets-byte committed:
Minplayers

Pockets-byte committed:
fixes test issues

distro loop, soda machine, etc

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte committed:
xenobio fix

Pockets-byte committed:
fuel in the tank

Pockets-byte committed:
massively expanded maint

Pockets-byte committed:
decreases min pop to 12 like other mid-pop ships

Pockets-byte committed:
reduces the laserspam

Pockets-byte committed:
Merge branch 'Robot-Ship' of https://github.com/Pockets-byte/NSV13 into Robot-Ship

Pockets-byte committed:
mostly continuous maint now

Pockets-byte committed:
Stormdrive, a few marking fixes, and flags that aren't under walls anymore

Pockets-byte committed:
desc

Pockets-byte committed:
Ooops

turfs and reactor ID

Pockets-byte committed:
work please

Pockets-byte committed:
banners

Pockets-byte committed:
more mice food

Pockets-byte committed:
cleaned up wiring slightly

Pockets-byte committed:
fixes some wiring oversights

bluewasabi24 commented:
![image](https://github.com/user-attachments/assets/3a876ef1-82c0-47d9-bc07-f955e0f43853)
Crates should not be spawning inside of the shuttle doors. Doors on the shuttle should also be shutters, not blast doors, or at least cargo access shuttle doors.

Pockets-byte committed:
fixes a little OOPS in tcoms

bluewasabi24 commented:
![image](https://github.com/user-attachments/assets/9501366b-35ef-4088-98c5-b1fe9d39c705)
Definitely don't need the AGCNR consoles if you're not gonna have it on the ship lol

Pockets-byte commented:
> ![image](https://private-user-images.githubusercontent.com/40776129/411268576-9501366b-35ef-4088-98c5-b1fe9d39c705.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkwNDk2NTcsIm5iZiI6MTczOTA0OTM1NywicGF0aCI6Ii80MDc3NjEyOS80MTEyNjg1NzYtOTUwMTM2NmItMzVlZi00MDg4LTk4YzUtYjFmZTlkMzljNzA1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA4VDIxMTU1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZkMWE4OWVkOGU0NTY5ZTdhNWVjMWM5N2EzNGNkNjNkNDA0MjRlOTk5ZjI4ZDIxMjE5Nzk5MDU2MDMwYjlmNDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.STEFZ3iT3TWdCGz3SDJG4etprcDAIiR5Btsu7nNi0yc) Definitely don't need the AGCNR consoles if you're not gonna have it on the ship lol

those are the gas tank consoles- I didn't feel the need to change them

bluewasabi24 commented:
![image](https://github.com/user-attachments/assets/ec533e40-290d-4a6c-819e-aad2000458fa)
The air distro pipes spawn above the carpet, at least in engineering.

Pockets-byte committed:
no beno

Pockets-byte committed:
agcnr return, gas miners gone

Dovahkitt3 commented:
![image](https://github.com/user-attachments/assets/9c154a82-be3e-4568-811e-ad75ba817e2f)
Doors are not linked to button.

Map is missing fax machines for multiple departments.

Omppusolttu commented:
All the fulltile firelocks have a tendency to close very fast after they are opened, but only sometimes.

To be fair I think the window firelocks also do this when the server is under strain, but the consiquence of that isn't getting crushed.

bluewasabi24 commented:
Admins politely requesting for fax machines, as well.

Pockets-byte committed:
post firelock NSV

Pockets-byte committed:
fixed a lot of my big screwups... I hope

Pockets-byte committed:
standard atmos layout

Pockets-byte committed:
pipes

Pockets-byte committed:
more fax machines. fixed cargo shuttle

BetaCentari commented on _maps/vonneumann.json:
```diff
@@ -0,0 +1,35 @@
+{
+	"map_name": "SGV Von Neumann Mark III",
```
Two things: 
The name of the ship is a big lengthy to go on the voting board and doesn't follow the ship naming standard. Shortening the title of the ship to SGV Von Neumann for the map_name section in the .json file should be just fine and you can keep the Mark III in the map_description section. 

If we can change the description from having references to civilian class or civilian grade to something to indicate standard grade, that'd make a bit more sense as SolGov is a military and governing force. Some examples: 

-Remove civilian from point defense fridge so it reads "Von Neumann Class Mark III Point Defense Fridge"
-Civilian Grade Hull could read "Standard Grade Hull" 
-Civilian grade defensivei armament could read "Disproportionately large armament for a point defense vessel"

Pockets-byte replied:
sure ok

Pockets-byte replied:
how about light hull, since standard grade doesn't really say anything

Pockets-byte committed:
lore

Pockets-byte committed:
desc tweaks

Pockets-byte committed:
the cloner is REAL

someone543 commented:
![BotsPatrol](https://github.com/user-attachments/assets/4bf3153d-a692-4aee-addb-cbe2028c5699)

Bots on patrol get stuck and only patrol from one end of this hallway to the other

![Screenshot 2025-02-09 104051](https://github.com/user-attachments/assets/0ba64c45-2150-49d1-8512-86a38918d130)

Miners don't have access to open the welding tool locker. Mining equipment is also missing a oxygen tank storage

![Screenshot 2025-02-09 104155](https://github.com/user-attachments/assets/49832f25-5e74-483b-8796-a85379312de2)

Mining hangar airlock pressurizes going to space, and depressurizes when going inside

Pockets-byte committed:
fixes robots, airlocks

Pockets-byte commented:
> ![BotsPatrol](https://private-user-images.githubusercontent.com/95106800/411291898-4bf3153d-a692-4aee-addb-cbe2028c5699.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkxMzQyNzAsIm5iZiI6MTczOTEzMzk3MCwicGF0aCI6Ii85NTEwNjgwMC80MTEyOTE4OTgtNGJmMzE1M2QtYTY5Mi00YWVlLWFkZGItY2JlMjAyOGM1Njk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA5VDIwNDYxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1ZTRmOWM4Y2I3ZTRmNmQ2NTZhNzcxNzc4YjU0YzNiOTdmZjQxOTFkMjc0N2MzN2ExZmYyY2I5M2EwMmVkNmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.q2w3q7AXF0e9Mgcd_mCAssVP-2KGr0JnSj0Jx5k6Crw)
> 
> Bots on patrol get stuck and only patrol from one end of this hallway to the other
> 
> ![Screenshot 2025-02-09 104051](https://private-user-images.githubusercontent.com/95106800/411292059-0ba64c45-2150-49d1-8512-86a38918d130.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkxMzQyNzAsIm5iZiI6MTczOTEzMzk3MCwicGF0aCI6Ii85NTEwNjgwMC80MTEyOTIwNTktMGJhNjRjNDUtMjE1MC00OWQxLTg1MTItODZhMzg5MThkMTMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA5VDIwNDYxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUyZTg4OGQxYTI3MDUyOTc2ZTc5NmIzZjlhMWM0Mjc5MDgxYmZhMGI4NzMxNmY2OTExODk1MDMzYTVmMmU2ZWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.AqCerfIRVedp7yH0zXp4nD928VsPE1dkJCWXw3FUn7E)
> 
> Miners don't have access to open the welding tool locker. Mining equipment is also missing a oxygen tank storage
> 
> ![Screenshot 2025-02-09 104155](https://private-user-images.githubusercontent.com/95106800/411292086-49832f25-5e74-483b-8796-a85379312de2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkxMzQyNzAsIm5iZiI6MTczOTEzMzk3MCwicGF0aCI6Ii85NTEwNjgwMC80MTEyOTIwODYtNDk4MzJmMjUtNWU3NC00ODNiLTg3OTYtYTg1Mzc5MzEyZGUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA5VDIwNDYxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI0Y2VjNDdmYTczZjZiNzdkOWE2NzRkZTU3MDIxNWZkMTUzZWQwYjhlMjMwZDAxZGI5ODdjZWZmMjhmMDJmZjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.43IPJEEXFxyTePE5zx7iv07uKC0CHu-c_nl9vf71buQ)
> 
> Mining hangar airlock pressurizes going to space, and depressurizes when going inside

Should be fixed now… I hope

Pockets-byte committed:
atmos redesign

couldn't test, technical issues on my side

Pockets-byte committed:
Update vnmk3.dmm

Pockets-byte committed:
removes mix valves

Pockets-byte committed:
atmoso

Pockets-byte commented:
> ![image](https://private-user-images.githubusercontent.com/40776129/411268061-3a876ef1-82c0-47d9-bc07-f955e0f43853.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkxNjc0MTYsIm5iZiI6MTczOTE2NzExNiwicGF0aCI6Ii80MDc3NjEyOS80MTEyNjgwNjEtM2E4NzZlZjEtODJjMC00N2Q5LWJjMDctZjk1NWUwZjQzODUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjEwVDA1NTgzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyNDI1NDY3ODcwZGQ1MDFhMzAxNjAwNDY1NjcyYWEyOGMwYmJmYTU2NDA2MzMwYWFmZWI2ZjJjNGIxMTg4MWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.e1ry7FVJg9-ZrvJ8k2QEBxTXPZ6TVPeIDaclhOmwBGM) Crates should not be spawning inside of the shuttle doors. Doors on the shuttle should also be shutters, not blast doors, or at least cargo access shuttle doors.

this is unavoidable with a small shuttle- it spawns crates on all open tiles, so short of making the shuttle bigger, I can't fix this

Pockets-byte committed:
fixes cargo shuttle, makes maze slightly better/worse

Pockets-byte committed:
might make ship slightly less dodgy

Pockets-byte committed:
the damage sprites work.

Bokkiewokkie commented:
> > ![image](https://private-user-images.githubusercontent.com/40776129/411268061-3a876ef1-82c0-47d9-bc07-f955e0f43853.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkxNjc0MTYsIm5iZiI6MTczOTE2NzExNiwicGF0aCI6Ii80MDc3NjEyOS80MTEyNjgwNjEtM2E4NzZlZjEtODJjMC00N2Q5LWJjMDctZjk1NWUwZjQzODUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjEwVDA1NTgzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyNDI1NDY3ODcwZGQ1MDFhMzAxNjAwNDY1NjcyYWEyOGMwYmJmYTU2NDA2MzMwYWFmZWI2ZjJjNGIxMTg4MWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.e1ry7FVJg9-ZrvJ8k2QEBxTXPZ6TVPeIDaclhOmwBGM) Crates should not be spawning inside of the shuttle doors. Doors on the shuttle should also be shutters, not blast doors, or at least cargo access shuttle doors.
> 
> this is unavoidable with a small shuttle- it spawns crates on all open tiles, so short of making the shuttle bigger, I can't fix this

Just using regular doors will fix it, because they close automatically. The cargo shuttle *dock* also does not need tiny fans, just having some doors and directional firelocks there is fine enough. (try to avoid tiny fans in mapping)

An unrelated observation. Please make the blue decal line point to Medical, or otherwise install a green line to do so. Command is not really something people struggle (or have a priority) to find, and medical is pretty much the single most important thing to be able to easily find.

Pockets-byte committed:
minor floor marking changes and no cargo shuttle tinyfans

Pockets-byte commented:
> > > ![image](https://private-user-images.githubusercontent.com/40776129/411268061-3a876ef1-82c0-47d9-bc07-f955e0f43853.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkxNjc0MTYsIm5iZiI6MTczOTE2NzExNiwicGF0aCI6Ii80MDc3NjEyOS80MTEyNjgwNjEtM2E4NzZlZjEtODJjMC00N2Q5LWJjMDctZjk1NWUwZjQzODUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjEwVDA1NTgzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyNDI1NDY3ODcwZGQ1MDFhMzAxNjAwNDY1NjcyYWEyOGMwYmJmYTU2NDA2MzMwYWFmZWI2ZjJjNGIxMTg4MWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.e1ry7FVJg9-ZrvJ8k2QEBxTXPZ6TVPeIDaclhOmwBGM) Crates should not be spawning inside of the shuttle doors. Doors on the shuttle should also be shutters, not blast doors, or at least cargo access shuttle doors.
> > 
> > 
> > this is unavoidable with a small shuttle- it spawns crates on all open tiles, so short of making the shuttle bigger, I can't fix this
> 
> Just using regular doors will fix it, because they close automatically. The cargo shuttle _dock_ also does not need tiny fans, just having some doors and directional firelocks there is fine enough. (try to avoid tiny fans in mapping)
> 
> An unrelated observation. Please make the blue decal line point to Medical, or otherwise install a green line to do so. Command is not really something people struggle (or have a priority) to find, and medical is pretty much the single most important thing to be able to easily find.

Done ✔️

Pockets-byte committed:
added smooth variant sprites for guidelines and the solgov roundel. used them on the ship

help.

Pockets-byte committed:
icon changed?

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
adds oxygen tanks to mining

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Merge remote-tracking branch 'upstream/master' into Robot-Ship

Pockets-byte committed:
clears marge conflicts

Pockets-byte committed:
Dumb wiring mistakes fixed

Pockets-byte committed:
minor changes.

a few missing lightswitches, a lamp

Pockets-byte committed:
few dumb things that I fixed before it crashed

Pockets-byte committed:
more small mistakes

Pockets-byte committed:
few cable changes

Pockets-byte committed:
Merge remote-tracking branch 'upstream/master' into Robot-Ship

Pockets-byte committed:
few fixes

button for the incinerator. layering

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
enlarges atmos tanks. fixes a few area fuckups.

Bokkiewokkie reviewed (changes requested):
Things I noted while flying around the map the other day:
The bottom left and bottom right corners of maintenance have diagonally unconnected walls (exposed to space, even), please connect these in some way (grilles don't count).
The XO's quarters should probably have a light a bit further on the right side of it, that corner can be quite dark.
The photocopier in there is also a full machine, so you can remove the table (or give them an XO fax machine?
Tech storage contains an omnilathe circuitboard, please remove that.
It also seems like the tech storage contains both secure and normal boards in it, so please give it a vault door (and move the camera from the wall with an APC to a free one). Adding a space gap between it and maintenance also helps with increasing security.
There are still several cameras on top of other wall mounts. While technically not required, they look better when put on any of the rest of the free wall. (some locations are: the reactor room, northeast section of halway under dorms, xenobio, and EVA storage).
The overmap viewscreen in the nanite lab is under an air alarm, maybe put it in xenobio or just remove the viewscreen.
It may be nice to move the fax machine or clothes vendor to the bottom right corner of RnD, since that's still empty and the rest of the room is a bit more crowded.
There's a status display behind the medical cryotube, making it hard to see (maybe put it on the wall above the cloner?)
I also have a suggestion to make dorms/the kitchen a bit more accommodating, at the cost of a random room:
![image](https://github.com/user-attachments/assets/3712099c-ed1a-4199-9a2e-53d21337a6fe)
There are a lot of lamps in the hangar and genetics. it almost feels too bright.

Pockets-byte commented:
> Things I noted while flying around the map the other day: The bottom left and bottom right corners of maintenance have diagonally unconnected walls (exposed to space, even), please connect these in some way (grilles don't count). The XO's quarters should probably have a light a bit further on the right side of it, that corner can be quite dark. The photocopier in there is also a full machine, so you can remove the table (or give them an XO fax machine? Tech storage contains an omnilathe circuitboard, please remove that. It also seems like the tech storage contains both secure and normal boards in it, so please give it a vault door (and move the camera from the wall with an APC to a free one). Adding a space gap between it and maintenance also helps with increasing security. There are still several cameras on top of other wall mounts. While technically not required, they look better when put on any of the rest of the free wall. (some locations are: the reactor room, northeast section of halway under dorms, xenobio, and EVA storage). The overmap viewscreen in the nanite lab is under an air alarm, maybe put it in xenobio or just remove the viewscreen. It may be nice to move the fax machine or clothes vendor to the bottom right corner of RnD, since that's still empty and the rest of the room is a bit more crowded. There's a status display behind the medical cryotube, making it hard to see (maybe put it on the wall above the cloner?) I also have a suggestion to make dorms/the kitchen a bit more accommodating, at the cost of a random room: ![image](https://private-user-images.githubusercontent.com/43698041/447335571-3712099c-ed1a-4199-9a2e-53d21337a6fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDgyMTIyMzMsIm5iZiI6MTc0ODIxMTkzMywicGF0aCI6Ii80MzY5ODA0MS80NDczMzU1NzEtMzcxMjA5OWMtZWQxYS00MTk5LTlhMmUtNTNkMjEzMzdhNmZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTI1VDIyMjUzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmMzlmOWIxZDA3MmNjMDY5NWJhNTIzMzAzNWRjYmEwMmMxODMyNjdkMGJmYjYwZTZkNjMzYmViMWQwZGU1MTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.u5scqD_vTQrDTayrK3i9ZjhnTnjflIbZfN4xhwb4e20) There are a lot of lamps in the hangar and genetics. it almost feels too bright.

I'm actually kinda happy with dorms right now, and I'm loathe to remove more maint
I'm looking at it though

Pockets-byte committed:
some suggested fixes

Pockets-byte committed:
camera camera camera camera

Bokkiewokkie commented:
I've untestmerged this because it conflicts with weapon datums right now

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
total service destruction

Pockets-byte committed:
reconstruction

Pockets-byte committed:
bervice

Pockets-byte committed:
the thing

Pockets-byte committed:
fixed fat fingered floors

Pockets-byte committed:
fixes a dumb

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
updates AI vncs to have working weapons

and hellfires because I'm mean

Pockets-byte committed:
replaces hellfires with a custom torpedo

the VNC now gets it's own variant of the shredder with higher damage and less armor pen as a balance between hellfires and normal torps

Pockets-byte committed:
solgov weapon lockers

Pockets-byte committed:
bureaucracy update

paper is everywhere now

Pockets-byte committed:
more maint secrekts and a few other changes I've already forgotten about

Pockets-byte committed:
another maint random room

Pockets-byte committed:
not private security

Pockets-byte committed:
nukes the nano-air alarm. also maybe fixes one of the gun cases

Pockets-byte committed:
gives the other heads ptsd

uncertain about this change honestly

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Increases the vn’s max pop to 40, because size is an illusion or something

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

red031000 commented:
Hello
I noticed while on round that a floorbot would repeatedly go between two locations several times, and never hit any other patrol nodes, I have attached images of the locations, this being, just outside atmos and just outside engineering
<img width="249" height="202" alt="image" src="https://github.com/user-attachments/assets/a8daa751-51c8-4deb-b8d6-f5b8d1974823" />
<img width="470" height="370" alt="image" src="https://github.com/user-attachments/assets/b6e3184f-8853-41e0-9385-480f0ecb663c" />

Pockets-byte committed:
Merge branch 'BeeStation:master' into Robot-Ship

Pockets-byte committed:
Buffs(nerfs?) the VNC’s durability because I lived please nerf

Pockets-byte committed:
Nerfs the torpedo spam from solgov slightly

Pockets-byte committed:
bots no worky.

Pockets-byte committed:
sprite-changes

Pockets-byte committed:
fixes robots maybe

Pockets-byte committed:
rfooled hull.

Pockets-byte committed:
fixes overlap
